### PR TITLE
HIP-000 | Prasanth, Veena | Fixes date issue in hip

### PR DESCRIPTION
**Pull request by Veena-tw**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs [hunk 1]
```diff
@@ -100,7 +100,7 @@ await httpClient.SendAsync(CreateHttpRequest(dataPushUrl, dataResponse, accessTo
 
             await dataFlowNotificationClient.NotifyCm(consentMangerUrl,
                 new DataNotificationRequest(dataResponse.TransactionId,
-                    DateTime.Now,
+                    DateTime.UtcNow,
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs:103]: does it fix automatically?

Comment (ganesanarun) [reply src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs:103]: We had string and all the DNR class?

File: src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs [hunk 2]
```diff
@@ -71,7 +72,8 @@ public DataEntryFactory()
                 });
             }
 
-            var keyStructure = new KeyStructure(DateTime.Now.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ"),
+            var dto = DateTimeOffset.Now;
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs:75]: Do we need to have local variable dto? can we directly use DateTimeOffset.Now.LocalDateTime only?

Comment (Veena-tw) [reply src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs:75]: yes, will change

File: src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs (hunk 1 above)
Comment (Veena-tw) [reply src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs:103]: no this fixes automatically, tested it.

**Pull request closed without merging**